pkg/gather/clusterconfig/openshift_apiserver_operator_logs_test.go

```diff
@@ -0,0 +1,44 @@
+package clusterconfig
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/RedHatInsights/insights-operator-utils/tests/helpers"
+	"github.com/stretchr/testify/assert"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kubefake "k8s.io/client-go/kubernetes/fake"
+)
+
+func TestGatherOpenShiftAPIServerOperatorLogs(t *testing.T) {
+	const testPodNamespace = "openshift-apiserver-operator"
+	// there's no way to specify logs fake pod will have, so we can only search for a hardcoded string "fake logs"
+	const stringToSearch = "fake logs"
+
+	coreClient := kubefake.NewSimpleClientset().CoreV1()
+	ctx := context.Background()
+
+	_, err := coreClient.Pods(testPodNamespace).Create(
+		ctx,
+		&corev1.Pod{ObjectMeta: metav1.ObjectMeta{
+			Name: testPodNamespace,
+		}},
+		metav1.CreateOptions{},
+	)
+	helpers.FailOnError(t, err)
```

Comment:
I am asking myself. Is it worth of adding this new dependency because of using one function in the tests?

Reply:
and I tend to say: No it's not :)

Reply:
Well, there are some other functions https://github.com/RedHatInsights/insights-operator-utils/tree/master/tests/helpers which we may need in the future, but I can remove it, no problems

Reply:
When looking at the change in `go.sum` (toooo many new stuff) then I would remove it. I would keep the unit tests as simple as possible (we can improve the integration tests once we have the independent CI image). Thanks and sorry.